### PR TITLE
Add otmorris

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/otmorris/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+if "%ARCH%"=="32" (set CPU_ARCH=i686) else (set CPU_ARCH=x86_64)
+%PKG_NAME%-%PKG_VERSION%-py%PY_VER%-%CPU_ARCH%.exe /userlevel=1 /S /FORCE /D=%PREFIX%
```
Comment (jakirkham) [inline recipes/otmorris/bld.bat]: Can we not build this on Windows too? Seems like we are using `cmake`, which should make this very easy.

Comment (jschueller) [inline recipes/otmorris/bld.bat]: @jakirkham the problem is building openturns with msvc; we only succeeded on building for vc10, without bindings. Also the build time would be a problem.

**Comment by jschueller** [comment]
@ocefpaf this one too ?

File: recipes/otmorris/build.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DUSE_SPHINX=OFF \
+  ..
+
+make install -j${CPU_COUNT}
+DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib ctest -R pyinstallcheck --output-on-failure -j${CPU_COUNT}
```
Comment (ocefpaf) [inline recipes/otmorris/build.sh]: I am not going to ask about `ctest` this time :smile:

Comment (jschueller) [inline recipes/otmorris/build.sh]: @ocefpaf : ]

**Pull request merged**